.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: go
+go:
+- 1.8
+
+go_import_path: github.com/dachad/check-max-tcp-connections
+
+script: script/cibuild
```

Comment:
Please, configure your editor to add the EOL to the last line, please

Reply:
done